family2family/build.gradle

```diff
@@ -26,7 +30,7 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
     compile("org.springframework.boot:spring-boot-starter-data-rest")
-    compile("org.springframework.security.oauth:spring-security-oauth2:${springOauthVersion}")
+    compile("org.springframework.security.oauth:spring-security-oauth2")
```

Comment:
@toscott Without the explicit version here, we got an upgrade to 2.0.13-RELEASE.